src/components/Counter/docs/index.story.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { Counter } from '../';
+import Examples from './examples';
+import {
+  header,
+  api,
+  divider,
+  importExample,
+  playground,
+  tab,
+  code as baseCode,
+  tabs,
+  testkit,
+  title,
+} from 'wix-storybook-utils/Sections';
+import { allComponents } from '../../../../stories/utils/allComponents';
+
+const code = config =>
+  baseCode({ components: allComponents, compact: true, ...config });
+
+export default {
+  category: 'Components',
+  storyName: 'Counter',
+  component: Counter,
+  componentPath: '../Counter.tsx',
+  componentProps: () => ({
+    'data-hook': 'storybook-Counter',
+    onChange: (val: string) => console.log(val),
+    value: 15,
+    step: 1,
+    max: 20,
+    min: 10,
+    error: false,
+    disabled: false
+  }),
+  exampleProps: {
+    value: 15,
+    step: 1,
+    max: 20,
+    min: 10,
+    error: false,
+    disabled: false
+  },
+  dataHook: 'storybook-Counter',
+  sections: [
+    header(),
+    tabs([
+      tab({
+        title: 'Usage',
+        sections: [
+          importExample({
+            source: `import { Counter } from 'wix-ui-tpa/Counter';`
+          }),
+
+          divider(),
+
+          title('Examples'),
+
+          ...[{ title: 'Example', source: `import { Counter } from 'wix-ui-tpa/Counter';` }].map(code),
+        ],
+      }),
+
+      ...[
+        { title: 'API', sections: [api()] },
+        { title: 'TestKit', sections: [testkit()] },
+        { title: 'Playground', sections: [playground()] },
+      ].map(tab),
+    ]),
+  ],
+  examples: <Examples />,
```

Comment:
Use `settingsPanel` for the example.
You can see how we wrote `Card/docs/index.story.tsx`, that should be a good reference.

Reply:
done